### PR TITLE
remove need for renaming

### PR DESCRIPTION
**Pull request by mr-c**

### CODE REVIEW

File: steps/wsclean.cwl [hunk 1]
```diff
@@ -50,27 +50,29 @@ arguments:
  - $(inputs.size_x)
  - $(inputs.size_y)
  - prefix: -temp-dir
-   valueFrom: /tmp
```
Comment (gijzelaerr) [inline steps/wsclean.cwl:53]: this is broken on OSX https://github.com/common-workflow-language/cwltool/issues/682

**Comment by mr-c** [comment]
This works for me, except that `tigger-restore` doesn't seem to be part of the Docker image yet.

**Comment by gijzelaerr** [comment]
So this works in the case of WSClean since it has a filename prefix flag, but for other tools (like simms) it would require the introduction of a new prefix variable. To me this really feels like something that should be handled by CWL.

**Comment by mr-c** [comment]
I agree, just wanted to show you what is possible today.

**Comment by gijzelaerr** [comment]
ok thanks ;)

**Review by gijzelaerr — CHANGES_REQUESTED** [review]

File: steps/wsclean.cwl (hunk 1 above)
Comment (mr-c) [reply steps/wsclean.cwl:53]: Bummer. I'll back it out then.

Comment (mr-c) [reply steps/wsclean.cwl:53]: Fixed

**Comment by mr-c** [comment]
Ready for merge

**Comment by gijzelaerr** [comment]
This is incorrect, it results in:
```
skymodel.fits
skymodel_0.txt
skymodel_0_meerkat_0h10s.MS-dirty.fits
skymodel_0_meerkat_0h10s.MS-image.fits
skymodel_0_meerkat_0h10s.MS-model.fits
skymodel_0_meerkat_0h10s.MS-residual.fits
skymodel_1.txt
skymodel_10.txt
skymodel_10_meerkat_0h10s.MS-dirty.fits
skymodel_10_meerkat_0h10s.MS-image.fits
skymodel_10_meerkat_0h10s.MS-model.fits
skymodel_10_meerkat_0h10s.MS-residual.fits
skymodel_1_meerkat_0h10s.MS-dirty.fits
skymodel_1_meerkat_0h10s.MS-image.fits
skymodel_1_meerkat_0h10s.MS-model.fits
skymodel_1_meerkat_0h10s.MS-residual.fits
```
while we expect:
```
1-skymodel.fits
1-skymodel.txt
1-wsclean-dirty.fits
1-wsclean-image.fits
1-wsclean-model.fits
1-wsclean-residual.fits
2-skymodel.fits
2-skymodel.txt
2-wsclean-dirty.fits
2-wsclean-image.fits
2-wsclean-model.fits
2-wsclean-residual.fits
```

The `skymodel.fits` file is overwritten every step. Anyway, the whole point is not that we don't know how to fix this, it was working.

**Pull request merged**